server/models/index.js

```diff
@@ -1,12 +1,98 @@
 const conn = require('./conn');
-const LineItem = require('./LineItem');
+//const LineItem = require('./LineItem');
 const Order = require('./Order');
 const Product = require('./Product');
 const User = require('./User');
 
-//DB Relations?
-//SEED?
+Order.belongsTo(User)
+User.hasMany(Order)
 
-const sync = () => { conn.sync({ force: true }) };
+// Order.hasMany(LineItem)
+// LineItem.belongsTo(Order)
+// LineItem.belongsTo(Product)
 
-module.exports = { sync }
+const syncAndSeed = () => conn.sync({ force: true })
```

Comment:
(Testing inline comments) We might need to set the sync to false when we deploy the app, we want to keep entered data in the database
```suggestion
const syncAndSeed = () => conn.sync({ force: false })
```

Reply:
Wow this is pretty cool feature :)